### DISCUSSION TITLE
ALFA AWUS1900

### DISCUSSION THREAD

**Discussion by Tockman** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#87] [Q&A]
As far as I remember this model was in the list of supported. But since some moment I can't see it there. Did it prove incompatibility with Big Sur/Monterey or there's some other reason? It's built on RTL8814AU chipset so I wonder if some problems could appear.

**Comment by chris1111** [comment]
Try with this?
https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/files/6759030/Wireless.USB.OC.Big.Sur.Adapter-V7.zip

**Reply by Tockman** [reply]
First I have to decide if I'm buying it or not ;) And it depends on does it work with your drivers set or no.

**Reply by Tockman** [reply]
[Here](https://www.youtube.com/watch?v=GC5rx5UfMaA) they say that 1900 will go.

**Comment by chris1111** [comment]
Ha OK 😆 Yes it work

**Comment by chris1111** [comment]
Probably will work with this [Special Program Wireless USB OC Big Sur Adapter-V8.zip](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/files/6881112/Wireless.USB.OC.Big.Sur.Adapter-V8.zip)


I commit an official  Release v8 today  and create new special program 
https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/commit/70dc4367393ac31d87a8487f240ee0dd54706792


[Official V8  ](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/releases/tag/V8)

**Reply by Tockman** [reply]
Thank you! I'll try it.

**Reply by Tockman** [reply]
By the way, I've just noticed that came to this discussions page by mistake. I was on "Wireless USB Big Sur Adapter" project page (not OC!) and press on "Report working adapter" link. And that link lead me there.

**Comment by chris1111** [comment]
Read the page Working adapter the special program is also add on  [Wireless USB Big Sur Adapter](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter)

**Reply by Tockman** [reply]
Thank you Chris.

**Reply by chris1111** [reply]
> Thank you Chris.


Let me know if its work
